kvmagent/kvmagent/plugins/port_mirror_plugin.py

```
@@ -158,17 +161,41 @@ def _clear_mirror_dst_config(self, bridge_name, device_name,mirror_device_name):
 
+
+
+    def _get_physical_target_device(self, mirror):
+        """获取物理机目标设备名称"""
+        if mirror.dstEndPointType == "UpLinkPort":
+            # 镜像到物理机网卡
+            if mirror.interfaceName:
+                if mirror.vlanId == 0:
+                    return mirror.interfaceName
+                else:
+                    return "%s.%d".format(mirror.interfaceName, mirror.vlanId)
```

Ruff (0.14.1)

`.format` call has unused arguments at position(s): 0, 1

Remove extra positional arguments at position(s): 0, 1

(F523)